IntelliTect.TestTools.TestFramework/ExampleTests/ExampleTests/ExampleTests.csproj

```
@@ -2,16 +2,29 @@
 
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
-
+    <LangVersion>9.0</LangVersion>
```

```suggestion
    <LangVersion>11.0</LangVersion>
```
Can we bump to latest lang version?